Mathlib/Algebra/Hom/Equiv/Basic.lean

```diff
@@ -201,12 +201,16 @@ variable [Mul M] [Mul N] [Mul P] [Mul Q]
 -- Porting note: `to_fun_eq_coe` no longer needed in Lean4
 #noalign mul_equiv.to_fun_eq_coe
 #noalign add_equiv.to_fun_eq_coe
```

Comment:
This note is also wrong, these are needed.

Reply:
The simp normal form is now `coe_toEquiv`

Reply:
Why isn't the simp-normal form `FunLike.coe`?

Reply:
Sorry, I was writing from mobile. I meant that the `simp` lemma we need is `coe_toEquiv`, not `toFun_eq_coe`, becase `e.toFun` is actually `e.toEquiv.toFun`, and `Equiv.toFun_eq_coe` simplifies it to `FunLike.coe e.toEquiv`.

Reply:
Anyway, I'm going to merge this now. We can fix more in another PR.